src/tests/unit/tests/DetailsView/components/should-show-report-export-button.test.ts

```diff
@@ -30,6 +32,20 @@ describe('ShouldShowReportExportButton', () => {
             .setup(m => m.getConfiguration(selectedTest))
             .returns(() => visualizationConfigurationMock.object);
         tabStoreData = { isChanged: false } as TabStoreData;
+
+        userConfigurationData = {
```

Comment:
This feels like a lot of implementation detail for this particular test to encode, I don't think we want to expect folks to find and update this stub every time a new user configuration field is added. Can this use a more contained stub?